src/pages/ProjectCreation/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Row, Col, Button, Space } from 'antd';
+import styles from './index.less';
+import PPContainer from '@/components/PPContainer';
+import { PlusOutlined } from '@ant-design/icons';
+import PPCard from '@/components/PPCard';
+import PPBlock from '@/components/PPBlock';
+
+const Project: React.FC = () => {
+  return (
+    <PPContainer>
+      <Row gutter={[20, 20]} style={{ marginTop: 101 }}>
```

Comment:
Since we reused this button here, maybe it will be reused at other placese, how about making a small component and import this button in 2 of these places?

Reply:
i think that's what you mean, add a button component